docker/Dockerfile

```diff
@@ -39,19 +29,13 @@ RUN \
     # OpenAI baselines
     libopenmpi-dev \
     # virtualenv
-    python3.5 \
-    python3.5-dev \
-    python3.5-tk \
-    python3.5-venv && \
+    python3 \
```

Comment:
can you pin this to python3.6, so we don't get caught off guard as before? i think deadsnakes has specific versions if apt doesn't

you could also consider just using pyenv to isolate yourself from Ubuntu.

Reply:
Yeah pyenv is a good idea. I am using that now.